src/pages/index.js

```diff
@@ -39,24 +64,36 @@ function handleImageClick(name, link) {
 const previewModal = new ModalWithImage("#preview__image-modal");
 
 // CREATE CARDS CONTAINER
+api
```

Comment:
Better to use `Promise.all` to load initial data

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all